examples/server-side-rendering/babel.config.js

```diff
@@ -23,6 +27,9 @@ module.exports = api => {
         },
       ],
     ],
-    plugins: ['@babel/plugin-syntax-dynamic-import', '@loadable/babel-plugin'],
+    plugins: [
+      '@babel/plugin-syntax-dynamic-import',
+      ['@loadable/babel-plugin', { target: api.caller(getTarget) }]
```

Comment:
the plugin needs to now if the target is node or browser environment, so the import method to use is require or native esm import